### DISCUSSION TITLE
ADD - Fooocus - mashb1t's 1-Up Edition

### DISCUSSION THREAD

**Discussion by bsdesigner** [LykosAI/StabilityMatrix#675] [Ideas]
Could you please add a very useful fork - Fooocus (mashb1t's 1-Up Edition): https://github.com/mashb1t/Fooocus

**Comment by ILikeToasters** [comment]
Would be a great addition since it gets the new features before the standard Fooocus.

**Comment by southbayjay** [comment]
Hey that would be fantastic if we could get his fork added in there! I use that fork over any of the others!

**Comment by dezvin** [comment]
I use it all the time too, because new features appear there first and really miss your model management functionality while working with it. Add its support please
